ffi/capi/src/bidi.rs

```diff
@@ -76,8 +77,9 @@ pub mod ffi {
             FnInStruct,
             hidden
         )]
-        #[diplomat::attr(not(any(dart, js)), disable)]
-        #[diplomat::attr(any(dart, js), rename = "for_text")]
+        // The only safe UTF-8 strings are those generated by the Diplomat layer in UTF-16 languages
+        #[diplomat::attr(supports = utf8_strings, disable)]
+        #[diplomat::attr(supports = utf16_strings, rename = "for_text")]
         pub fn for_text_valid_utf8<'text>(
```

Comment:
note: because the C backend respects `disable`, this method is hidden from it. If we want the C backend to be the ABI backend, we have to ignore `disable`.

Reply:
fine for now!